packages/esm-appointments-app/src/form/appointments-form.scss

```diff
@@ -0,0 +1,105 @@
+@use '@carbon/styles/scss/colors';
+@use '@carbon/styles/scss/spacing';
+@use '@carbon/styles/scss/type';
+
+$openmrs-background-grey: #ededed;
+
+.formWrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: calc(100vh - 6rem);
+}
+
+:global(.omrs-breakpoint-lt-desktop) .formWrapper {
+  background-color: $openmrs-background-grey;
+}
+
+:global(.omrs-breakpoint-gt-tablet) .formWrapper {
+  background-color: colors.$white;
+}
+
+.formGroup {
+  display: flex;
+  margin-bottom: spacing.$spacing-02;
+  padding: spacing.$spacing-05;
+
+  checkbox {
+    color: colors.$gray-70;
+    @include type.type-style('heading-compact-02');
+    margin-bottom: spacing.$spacing-03;
+  }
+}
+
+:global(.omrs-breakpoint-gt-tablet) .formGroup {
+  flex-direction: column;
+}
+
+:global(.omrs-breakpoint-lt-desktop) .formGroup {
+  flex-direction: row;
+
+  .heading {
+    flex: 1;
+  }
+
+  div {
+    flex: 3;
+  }
+}
+
+.heading {
+  color: colors.$gray-70;
+  @include type.type-style('heading-compact-02');
+  margin-bottom: spacing.$spacing-03;
+}
+
+.inputContainer {
+  display: flex;
+  flex-flow: row wrap;
+  gap: spacing.$spacing-05;
+  align-items: center;
+}
+
+.button {
+  height: 4rem;
+  display: flex;
+  align-content: flex-start;
+  align-items: baseline;
+  min-width: 50%;
+}
+
+.loader {
+  display: flex;
+  background-color: $openmrs-background-grey;
+  justify-content: center;
+  min-height: spacing.$spacing-09;
+  height: 100vh;
+}
+
+.tablet {
+  padding: spacing.$spacing-06 spacing.$spacing-05;
+  background-color: colors.$white;
+}
+
+.desktop {
+  padding: 0;
+}
+
+.weekSelect {
+  max-inline-size: fit-content;
+  padding-top: spacing.$spacing-03;
+}
+
+:global(.cds--select-input):focus {
```

Comment:
Hello @mogoodrich is this intended from the designs!, as this affects other fields not on the appointment form.
Thanks
cc @brandones @vasharma05

Reply:
It's a very good point @usamaidrsk ... I just copied this scss class over as-is from the patient-chart, but I would think that *everything* in this class should be namespaces in some way to only to apply to this page... and there's likely stuff that may not be needed at all.

Maybe the best thing to do would be to comment out everything from this class and view the appointments-form from the app and then *only* add back in things that are necessary for that appointment-form, namespacing as necessary.  How does that sound @usamaidrsk ?

Reply:
Thanks for the response @mogoodrich , I think we can try with the suggestion you've made.

Reply:
Yeah... the two @usamaidrsk flagged are a bit of a weird case.

```sass
.formWrapper :global(.cds--select-input):focus
```

Would be safe, but as-written, this is overriding the behaviour of all `.cds--select-input` elements.